tdrs-backend/tdpservice/parsers/schema_defs/tanf/t4.py

```diff
@@ -20,27 +20,31 @@
               Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
                     required=True, validators=[]),
               Field(item="4", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.month_year_yearIsLargerThan(1998),
+                                               validators.month_year_monthIsValid()]),
               Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.isAlphaNumeric(),
+                                               validators.notMatches('_'*11),
```

Comment:
is the `_`-fill worth adding to `util.value_is_empty`? `value_is_empty` is used internally for `required` fields